applications/metadata.js

```diff
@@ -52,6 +52,15 @@ var metadata = { "applications": [
         "jsfile": "vr-grabscale/VRBuildGrabScale.js",
         "icon": "vr-grabscale/logo.png",
         "caption": "VR SCALE"
+     },
+     {
+        "isActive": true,
+        "directory": "radar",
+        "name": "Radar",
+        "description": "Show where people are and teleport in the domain.",
```

Comment:
```suggestion
        "description": "Show where people are and teleport to them in the domain.",
```

Reply:
You can teleport to any position in the radar circle, or to an avatar. (But you can't teleport across domains, hence the "within the domain".)